bedrock/firefox/views.py

```diff
@@ -528,5 +528,8 @@ class HelloStartView(LatestFxView):
     def get_context_data(self, **kwargs):
         ctx = super(HelloStartView, self).get_context_data(**kwargs)
         incoming = self.request.GET.get('incomingConversation') or 'none'
+        source = self.request.GET.get('utm_content') or 'none'
+        source = source.replace('hello-tour_OpenPanel_', '')
```

Comment:
@alexgibson it's not wrong, and the template should handle it. I'm only saying that I think a trip to the server with this data doesn't add anything, and any movement of data from the user to the server is a potential attack vector. I'm also saying this as a backend person... so we're always worried about this kind of thing :)